shared/profile/user/index.tsx

```diff
@@ -371,7 +374,10 @@ class User extends React.Component<Props, State> {
           direction="vertical"
           fullWidth={true}
           fullHeight={true}
-          style={Styles.collapseStyles([styles.container, colorTypeToStyle(this.props.backgroundColorType)])}
+          style={Styles.collapseStyles([
+            {...styles.container, paddingTop},
```

Comment:
I think collapseStyles is supposed to avoid spreads, would go with a three elem array instead